Prace_domowe/Praca_domowa6/Grupa1/Kacper_Grzymkowski/PD6.ipynb

```diff
@@ -0,0 +1,1129 @@
+{
```

Comment:
<p>można było to opakować w funkcję, bo się powtarza</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>